webnn/resources/test_data/gemm.json

```diff
@@ -0,0 +1,1151 @@
+{
+  "tests": [
+    {
+      "name": "gemm with default options",
+      "inputs": {
+        "a": {
```

Comment:
Here `inputs` is a dictionary, it has **a** and **b** to align with two parameters of `gemm(MLOperand a, MLOperand b, optional MLGemmOptions options = {}); };`, the key 'a' and 'b' are used for input operands name, which would used to create input operands on https://github.com/BruceDai/wpt/blob/3edeb4c341e83794125804a06b8ba890a71bcc6b/webnn/gemm.https.any.js#L13